Inc/HALALMock/Services/Communication/UART/UART.hpp

```diff
@@ -151,7 +149,7 @@ class UART{
 	 *            successfully. Returns false if the UART is busy or a problem
 	 *            has occurred.
 	 */
-    static bool transmit_polling(uint8_t id, uint8_t data);
+    static bool transmit_polling(uint8_t id, uint8_t data){return true;};
```

Comment:
I dont know if this will compile with Warning as error, as this is unused-parameter